### PR TITLE
update test using c.Assert replace c.Fatal

### PR DESCRIPTION
**Pull request by Crazykev**
update test using c.Assert instead of c.Fatal  #16756 

Signed-off-by: Hao Zhang 21521210@zju.edu.cn

### CODE REVIEW

File: integration-cli/docker_cli_save_load_test.go [hunk 1]
```diff
@@ -129,37 +116,29 @@ func (s *DockerSuite) TestSaveImageId(c *check.C) {
...
+	c.Assert(err, checker.IsNil, check.Commentf("cannot set stdout pipe for grep: %v", err))
 
-	if err = tarCmd.Start(); err != nil {
-		c.Fatalf("tar failed with error: %v", err)
-	}
-	if err = saveCmd.Start(); err != nil {
-		c.Fatalf("docker save failed with error: %v", err)
-	}
+	err = tarCmd.Start()
+	c.Assert(err, checker.IsNil, check.Commentf("tar failed with error: %v", err))
```
Comment (HuKeping) [inline integration-cli/docker_cli_save_load_test.go]: c.Assert(tarCmd.Start(), checker.IsNil, check.Commentf("tar failed with error: %v", err))

File: integration-cli/docker_cli_save_load_test.go [hunk 2]
```diff
@@ -129,37 +116,29 @@ func (s *DockerSuite) TestSaveImageId(c *check.C) {
...
+	err = saveCmd.Start()
+	c.Assert(err, checker.IsNil, check.Commentf("docker save failed with error: %v", err))
```
Comment (HuKeping) [inline integration-cli/docker_cli_save_load_test.go]: c.Assert(saveCmd.Start(), checker.IsNil, check.Commentf("docker save failed with error: %v", err))

**Comment by Crazykev** [comment]
@HuKeping updated !

**Comment by cpuguy83** [comment]
LGTM

**Comment by estesp** [comment]
LGTM

**Pull request merged**